src/useSessionStorageValue/__docs__/story.mdx

```diff
@@ -59,6 +62,8 @@ function useSessionStorageValue<
   - **initializeWithValue** _`boolean`_ _(default: true)_ - Fetch storage value on first render. If
     set to `false` will make the hook yield `undefined` on first render and defer fetching of the
     value until effects are executed.
+  - **parse** _`function`_ - Custom parse function to use instead of JSON.parse
+  - **stringify** _`function`_ - Custom stringify function to use instead of JSON.stringify
```

Comment:
Better to include the type signatures of the functions here instead of just saying `function`.